app/Http/Controllers/Administration/DiagnosticsController.php

```diff
@@ -29,7 +28,7 @@ class DiagnosticsController extends Controller
 	 */
 	private function isAuthorized(): bool
 	{
-		return Gate::check(SettingsPolicy::CAN_SEE_DIAGNOSTICS, Configs::class) || AdminAuthentication::isAdminNotRegistered();
```

Comment:
I still think it is important to access the diagnostics if no admins are registered.